spark/src/main/scala/org/apache/comet/rules/CometExecRule.scala

```diff
@@ -668,7 +674,8 @@ case class CometExecRule(session: SparkSession) extends Rule[SparkPlan] {
         normalizePlan(plan)
       }
 
-      var newPlan = transform(normalizedPlan)
+      // FIXME: Should we move to separate Rule
+      var newPlan = transform(transformAndAddCopyExec(normalizedPlan))
```

Comment:
I am thinking of moving this transformation in sperate rule. But dont know where do i add this rule !!